projectgenerator/gui/generate_project.py

```diff
@@ -166,40 +166,41 @@ def accepted(self):
             if not self.ili_file_line_edit.text().strip():
                 if not self.ili_models_line_edit.text().strip():
                     self.txtStdout.setText(
-                        self.tr('Please set a valid INTERLIS model before creating the project.'))
+                        self.tr('\n\nPlease set a valid INTERLIS model before creating the project.'))
                     self.ili_models_line_edit.setFocus()
                     return
 
             if self.ili_file_line_edit.text().strip() and \
                     self.ili_file_line_edit.validator().validate(configuration.ilifile, 0)[0] != QValidator.Acceptable:
                 self.txtStdout.setText(
-                    self.tr('Please set a valid INTERLIS file before creating the project.'))
+                    self.tr('\n\nPlease set a valid INTERLIS file before creating the project.'))
                 self.ili_file_line_edit.setFocus()
                 return
 
         if self.type_combo_box.currentData() in ['ili2pg', 'pg']:
             if not configuration.host:
                 self.txtStdout.setText(
-                    self.tr('Please set a host before creating the project.'))
+                    self.tr('\n\nPlease set a host before creating the project.'))
                 self.pg_host_line_edit.setFocus()
                 return
             if not configuration.database:
                 self.txtStdout.setText(
-                    self.tr('Please set a database before creating the project.'))
+                    self.tr('\n\nPlease set a database before creating the project.'))
                 self.pg_database_line_edit.setFocus()
                 return
             if not configuration.user:
                 self.txtStdout.setText(
-                    self.tr('Please set a database user before creating the project.'))
+                    self.tr('\n\nPlease set a database user before creating the project.'))
                 self.pg_user_line_edit.setFocus()
                 return
         elif self.type_combo_box.currentData() in ['ili2gpkg', 'gpkg']:
             if not configuration.dbfile or self.gpkg_file_line_edit.validator().validate(configuration.dbfile, 0)[0] != QValidator.Acceptable:
                 self.txtStdout.setText(
-                    self.tr('Please set a valid database file before creating the project.'))
+                    self.tr('\n\nPlease set a valid database file before creating the project.'))
```

Comment:
These changes really should go in a separate pull request. It's ok to leave them this time, but please keep it in mind for next time.

Reply:
Thinking about this again, these are currently breaking all the translations, (and the `\n\n` part is not something a translator should have to take care off). This needs to be implemented differently, with the `\n\n` part outside the translated text)